Document/0x06b-Basic-Security-Testing.md

```diff
@@ -89,11 +89,11 @@ Once you've jailbroken your iOS device and Cydia has been installed (as shown in
 2. SSH into your iOS device.
   - The default users are `root` and `mobile`.
   - The default password is `alpine`.
-3. Change the default password for users `root` and `mobile`.
+3. Change the default password for both users `root` and `mobile`.
 4. Add the following repository to Cydia: `https://build.frida.re`.
 5. Install Frida from Cydia.
 
-Cydia allows you to manage repositories. One of the most popular repositories is BigBoss. If your Cydia installation isn't pre-configured with this repository, you can add it by navigating to Sources -> Edit, then clicking "Add" in the top left and entering the following URL:
+Cydia allows you to manage repositories. One of the most popular repositories is BigBoss (and one of the most popular software packages is the BigBoss Recommended Tools). If your Cydia installation isn't pre-configured with this repository, you can add it by navigating to Sources -> Edit, then clicking "Add" in the top left and entering the following URL:
```

Comment:
Perhaps you can rephrase it to: ydia allows you to manage repositories. One of the most popular repositories is BigBoss, which contains various packages, such as the BigBoss Recommended Tools package.

Reply:
Done.